### PR TITLE
docs: fix missing broken Node.js code snippets

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: docs/src/protractor-js.md [hunk 1]
```diff
@@ -98,26 +98,26 @@ Here's how to polyfill `waitForAngular` function in Playwright Test:
...
+        `);
+      }
+
+      await executeScriptAsync(page, clientSideScripts.waitForAngular, '');
     }
     ```
-    If you don't want to keep a version protractor around, you can also use this simpler approach using this function (only works for Angular 2+):
+    If you don't want to keep a version protractor around, you can also use this simpler approach using this function (only works for Angular 2+):/
     ```js
     async function waitForAngular(page) {
```
Comment (debs-obrien) [inline docs/src/protractor-js.md:122]: Why the / here?

File: docs/src/service-workers-experimental-network-events-js.md [hunk 2]
```diff
@@ -180,40 +180,41 @@ Consider the code snippets below to understand Playwright's view into the Reques
...
-      if (event.request.url.endsWith('tracker.js'))
-        return new Response('console.log('no trackers!')', {
-          status: 200,
-          headers: { 'Content-Type': 'text/javascript' },
-        });
-
-      // 4. Otherwise, fallthrough, perform the fetch and respond
-      return fetch(event.request);
-    })()
+        const response = await caches.match(event.request);
```
Comment (debs-obrien) [inline docs/src/service-workers-experimental-network-events-js.md:197]: Does indent here make sense?

Comment (mxschmitt) [reply docs/src/service-workers-experimental-network-events-js.md:197]: fixed

File: docs/src/protractor-js.md (hunk 1 above)
Comment (mxschmitt) [reply docs/src/protractor-js.md:122]: fixed, by accident.

**Pull request merged**